src/components/accessRequests/accessRequests.tsx

```diff
@@ -108,65 +153,34 @@ const AccessRequests = () => {
     );
   }
 
+  function parseDate(value: string) {
```

Comment:
Maybe it would help if we use a library such as "moment" for handling dates?

Reply:
It works fine right now so far. No need to library what you can code yourself 😁

Reply:
I put that onto the refactoring list.